src/api/python/_api.py

```
@@ -705,6 +705,54 @@ def release_xfer_handle(self, handle: nixl_xfer_handle):
+                        (addr, len, dev_id, remote_agent_name).
+           Returns the raw uintptr handle; pair with release_mem_view to free.
+
+    @param backends Optional list of backend names to limit the preparation to.
+    @return Opaque uintptr handle for the memory view.
+    """
+
+    @overload
+    def prep_mem_view(
+        self,
+        dlist: nixlBind.nixlXferDList,
+        backends: list[str] = [],
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

(B006)